docs/python/doxygenlib/cdParser.py

```diff
@@ -221,8 +226,7 @@ def parseDoxygenIndexFile(self, doxygen_index_file):
                     continue
                 if (kind == "dir"):
                     continue
-                if (kind == "file"):
-                    continue
```

Comment:
So I think this change is the heart of this PR - unfortunately, when we tested it, it had some negative side effects.
Specifically, many of our classes lost their docstrings after running `convertDoxygen.py` with this change.

One example was `pxr.Usd.ZipFileWriter` - in `Usd/__DOC.py`, we now get:

```python
   result["ZipFileWriter"].__doc__ = """"""
```

instead of:

```python
   result["ZipFileWriter"].__doc__ = """
Class for writing a zip file. This class is primarily intended to
support the .usdz file format. It is not a general-purpose zip writer,
as it does not implement the full zip file specification. However, all
files written by this class should be valid zip files and readable by
external zip modules and utilities.

"""
```

We're using Doxygen 1.8.18... which is old, but also newer than the 1.8.14 listed in [VERSIONS.md](https://github.com/PixarAnimationStudios/OpenUSD/blob/release/VERSIONS.md).

@chadrik, can you confirm whether or not you're seeing the same, and what version of Doxygen you're using?

Reply:
Hey @pmolodo , you made us realize we neglected to update VERSIONS.md for the addition of doxygen-awesome, which requires 1.9.x (we are using 1.9.6 internally).  We will be pushing out updates to VERSIONS.md shortly, but can you retry your experiments with a 1.9.x version?

Reply:
I tried a test with only removing these lines and comparing build output (using doxygen 1.8.x). The new __DOC.py appears to have duplicate (empty) __doc__ values for some things, like ZipFileWriter as @pmolodo notes, which end up overwriting the earlier defined __doc__ values.

Reply:
This is fixed.